subsys/bluetooth/controller/ll_sw/ll.c

```diff
@@ -194,6 +195,8 @@ int ll_init(struct k_sem *sem_rx)
 		    &_ticker_nodes[0], MAYFLY_CALLER_COUNT, &_ticker_users[0],
 		    RADIO_TICKER_USER_OPS, &_ticker_user_ops[0]);
 
+	ll_filter_reset(true);
```

Comment:
Move this after radio_init, just so that if in future ll_filter does make ctrl.c calls, ctrl.c has been initialised.